Themis/Views/Assessment/CorrectionSidebar/EditFeedback/AddFeedbackView.swift

```diff
@@ -15,6 +15,8 @@ struct AddFeedbackView: View {
     @Binding var showSheet: Bool
     
     var file: Node?
+    
+    let exercise: ExerciseOfSubmission?
```

Comment:
Why pass the whole `exercise` here when the list of `gradingCriteria` would be enough? I would rather try to pass down as little information as possible (reason for change request).

Reply:
Hmm, there's still the `exercise` here, no? I think this whole component only needs to get the `gradingCriteria`.